src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -2421,6 +2428,7 @@ def forward(
         mask_labels: Optional[Tensor] = None,
         class_labels: Optional[Tensor] = None,
         pixel_mask: Optional[Tensor] = None,
+        output_auxiliary_logits: Optional[bool] = None,
```

Comment:
This should probably go with a config argument (like `output_hidden_states` etc) that would give a default. Otherwise defaulting to `None` makes no sense.

Reply:
Ok, I made the change but will wait for @FrancescoSaverioZuppichini advice to make sure this is sound.

Reply:
Thanks! Just a little bit of context, the `auxiliary_logits` are consumed by the loss if `use_auxiliary_loss` was set to true so we could remove them from the output or add a flag in the config to return them

Reply:
So if someone uses both `output_auxiliary_logits=True` and `use_auxiliary_loss=True` then it wouldn't work currently is that it ?  (If so we need to change the consuming behavior into a copy I think)

Reply:
Friendly ping @FrancescoSaverioZuppichini

Reply:
It will work :)